### DISCUSSION TITLE
latency on lazer

### DISCUSSION THREAD

**Discussion by Zoraxus** [ppy/osu#17797] [Q&A] [answered]
I get around 0.5-0.6 ms on stable but now that I switched to lazer I get like 1.3-2.5 ms and I cant play because of it.
I don't know why it happened. I downloaded lazer and after importing everything I just had like 2.0 ms update every time I played.

**Comment by bdach** [comment, marked as answer]
are you using exclusive fullscreen? were you using compatibility mode on stable?

**Reply by Zoraxus** [reply]
I'm using fullscreen and no I'm not using compatibility

**Reply by bdach** [reply]
not borderless? are you sure?

most latency issues are due to the game not entering exclusive fullscreen. also i don't know where you're getting your millisecond figures from; if you're using draw FPS for those then that is not a correct comparison. lazer has a different threading structure and for input latency you want to be looking at the FPS of the input thread.

**Reply by Zoraxus** [reply]
yes im sure im on fullscreen and my input fps are the same as the fps i get on stable even a bit higher.

**Reply by Zoraxus** [reply]
what is the update meter thingy?

**Reply by integerrr** [reply]
ctrl+F11 twice
it looks like
<img width="544" alt="image" src="https://user-images.githubusercontent.com/7279624/163059116-a8e722fa-c8fd-4622-bf90-0fc5aaca9a8d.png">

**Reply by Zoraxus** [reply]
I know how it looks like but what does it mean? What is the difference between update and input? I'm asking that because update is one of the things that shoots up after I hit a note

**Reply by peppy** [reply]
To reiterate, if you "can't play" at 2-4ms it's not frame rate related, but something like exclusive full screen not working.

Are you on optimus hardware? When you hit win-g what happens?

**Reply by Zoraxus** [reply]
I'm not using optimus and hitting win-g does nothing. 
and its not that it's unplayable but when on stable i can stream like 200 bpm but on lazer i cant even stream 180 and im getting tons of 100s it just feels wrong as lazer is supposed to feel smoother

**Reply by Zoraxus** [reply]
after messing with stuff i found that when i use the in game tablet driver it works fine. for some reason using an external program for my tablet just messed everything up (i used the openTabletDriver and i have a ctl-472).
sorry for taking your time and thanks for the support.
